product-service/serverless.ts

```diff
@@ -33,7 +33,48 @@ const serverlessConfiguration: Serverless = {
       PG_DATABASE: process.env.PG_DATABASE,
       PG_USERNAME: process.env.PG_USERNAME,
       PG_PASSWORD: process.env.PG_PASSWORD,
+      SNS_TOPIC_ARN: { Ref: 'SNSTopic' }
     },
+    iamRoleStatements: [
+      {
+        Effect: 'Allow',
+        Action: ['sns:Publish'],
+        Resource: [{ Ref: 'SNSTopic' }]
+      }
```

Comment:
NOTE: IAM policy for SQS is **not needed**, because serverless framework automatically adds the needed policy itself - SQS is configured as event source for catalog-batch-process function